code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Dispenser.dm

```diff
@@ -377,13 +377,6 @@
 
 /datum/reagent/radium/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
 	M.apply_effect(10 * removed, IRRADIATE, blocked = 0) // Radium may increase your chances to cure a disease
-	if(M.is_diona())
-		M.adjustToxLoss(-20 * removed)
-		M.adjustBruteLoss(-20 * removed)
-		M.adjustFireLoss(-20 * removed)
-		if(!message_shown) // Not to spam message
-			to_chat(M, "<span class='notice'>You feel an extreme energy as your body regenerates faster.</span>")
-			message_shown = TRUE
```

Comment:
```
	if(M.is_diona())
		M.adjustToxLoss(-2 * removed)
		M.adjustBruteLoss(-2 * removed)
		M.adjustFireLoss(-2 * removed)
		if(!message_shown) // Not to spam message
			to_chat(M, "<span class='notice'>You feel an extreme energy as your body regenerates faster.</span>")
			message_shown = TRUE
```

I believe this is an okay compromise. I'll also attempt to present a code on making radium a muli-part chemical.